openapi.yaml

```diff
@@ -8916,3 +9072,101 @@ components:
       - $ref: '#/components/schemas/DeploymentAnalysis'
       description: Everything we know about the deployment.
       type: object
+    WorkTypeName:
+      description: Work type name. It is unique within the account scope.
+      minLength: 3
```

Comment:
@warenlg is that ok that the minimum length of the name is 3 symbols? What if someone wants to create a work type with the name `QA` for example?

Reply:
Why a min length is even needed?

Reply:
Why a min length is even needed?

Reply:
@vmarkovtsev ?

Reply:
Always include a min length everywhere - this is something I have learned over the last 1.5 years.
The questions to answer for the product are:
- Is it OK to have a work type with an empty name?
- Is it OK to have a work type with only one letter?
- ... 2 letters ...
- ... 3 letters ...

Reply:
Empty is not ok, but users should be able to name work types "A", "B" and "C" if they like. So I'd vote for a min length of 1 character.

Reply:
Vale